runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -162,6 +162,16 @@ default FairyRingMode swapFairyRing()
 		return FairyRingMode.LAST_DESTINATION;
 	}
 
+	@ConfigItem(
+		keyName = "swampGrandExchangeAbort",

```

Comment:
typo: "swampGrandExchangeAbort" -> "swapGrandExchangeAbort"

Reply:
😆 Thank you, must be watching too much Swampletics